### PR TITLE
change supervisor filter

### PR DESCRIPTION
**Pull request by tony-tran**

### CODE REVIEW

File: tests/Controller/EnMarche/ReferentControllerTest.php [hunk 1]
```diff
@@ -18,6 +18,7 @@
 /**
  * @group functional
  * @group referent
+ * @group wip
```
Comment (ottaviano) [inline tests/Controller/EnMarche/ReferentControllerTest.php]: :D 
try phpstorm to run one isolated test method :)

Comment (tony-tran) [reply tests/Controller/EnMarche/ReferentControllerTest.php]: Rohhh my bad, sorry.

Comment (thomasbisignani) [reply tests/Controller/EnMarche/ReferentControllerTest.php]: Another way, you can run it with :  

`docker-compose run --rm app vendor/bin/phpunit --filter ReferentControllerTest`

with the class / method name for the filter

File: templates/referent/users_list.html.twig [hunk 2]
```diff
@@ -150,7 +146,7 @@
                             Exemple: "59400, 18, CH"
                         </legend>
 
-                        <legend class="label__header label__header__interest">Centres d'intêret</legend>
+                        <legend class="label__header label__header__interest">Centres d'intêrêt</legend>
```
Comment (ottaviano) [inline templates/referent/users_list.html.twig]: `d'intérêt` ?

**Pull request merged**